### ISSUE TITLE
False positive with threading and daemon

### ISSUE THREAD

**Issue by pylint-bot** [PyCQA/pylint#268]
Originally reported by: **Claudiu Popa (BitBucket: [PCManticore](http://bitbucket.org/PCManticore), GitHub: @PCManticore)**

---

The following code should give a not-callable warning, but it doesn't, because inferring `a.daemon` will yield a YES node, thus ignoring the error.

```
#!python

import threading

class A(threading.Thread):
   @property
   def daemon(self):
       return False

a = A()
a.daemon(True)
```

---
- Bitbucket: https://bitbucket.org/logilab/pylint/issue/268
